### DISCUSSION TITLE
request: set/override default client cursor location

### DISCUSSION THREAD

**Discussion by williamla** [input-leap/input-leap#1724] [Ideas]
### What happened?

Request is to enable setting a default (or idle) client cursor location in the config. The cursor would essentially register at a designated or non-default location when the cursor isn't active on the client screen.

OR have the client remember the last location of the cursor and leave it as-is instead of recentering it each time.

When using barrier, the client system often centers the mouse to the middle of the client's screen after the mouse spends enough time on the primary/server screen. This ends up triggering a tooltip and in my case, a browser preview on the client/Windows machine (vertical screen, split with two browser windows ie top and bottom).

### Version

v2.4.0

### If applicable, where did you install Input Leap from?

n/a

### What OSes are you seeing the problem on? (Check all that apply)

Windows

### What OS versions are you using?

Windows 11 and macOS 13.

### Any other information

Basically the same as what's requested here: https://github.com/debauchee/barrier/issues/1708

**Comment by shymega** [comment]
This isn't a issue, but a feature request. Please use the 'Ideas' category from the Discussions tab in future.

Noted, feature request, PRs welcome.

**Reply by williamla** [reply]
Sorry about that. Thanks for the recategorization and consideration.

**Comment by solomongrundy6** [comment]
As the guy from the referenced barrier issue 1708, i'd love for this to be added too.  I'm currently using a hacky change to MSWindowsScreen.cpp I made for barrier to force it to instead go to 1920x1080 instead of the center of the screen.  This change would make it so I don't have to boot a Windows VM to build a custom version and just be able to use a stock input-leap, which would be great.
